### DISCUSSION TITLE
Corner seam hiding

### DISCUSSION THREAD

**Discussion by bavery79** [OrcaSlicer/OrcaSlicer#278] [Ideas]
Have you looked into SuperSlicer's additional seam location settings?  Coming from Cura, I've always felt PrusaSlicer was completely lacking in this regard, and pleasantly surprised to discover SuperSlicer's additional seam options to configure auto hiding seams at corners where possible.  Would absolutely love to see this sort of functionality ported over to BambuStudio!

**Comment by SoftFever** [comment]
`Hiding seams at corners` is always there in all slic3r-based slicers, just in under names.
When you choose `align` the seam, it will hide seams at corners.
